spec/injector_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'injector'
+
+describe Array do
+
+  it 'returns 8 when passed { |sum, n| sum + n } on [1, 5, 2]' do
+    expect([1, 5, 2].injector { |sum, n| sum + n }).to eq 8
+  end
+  
+  it 'returns "sheep" when passed do |memo, word| memo.length > word.length ? memo : word end on ["cat", "sheep", "bear"]' do
+    ['cat', 'sheep', 'bear'].injector do |memo, word|
+	  memo.length > word.length ? memo : word 
```

Comment:
Use 2 (not -1) spaces for indentation.<br>Tab detected.<br>Trailing whitespace detected.